Documentation/kubernetes-on-vagrant.md

```diff
@@ -60,6 +60,16 @@ Next, simply run `vagrant up` and wait for the command to succeed.
 Once Vagrant is finished booting and provisioning your machine, your cluster is good to go.
 
 ## Configure kubectl
+You can choose from one of the two following options.
+
+1. **KUBECONFIG path**
+
+```sh
+export KUBECONFIG="${KUBECONFIG}:$(pwd)/kubeconfig"
+kubectl config use-context vagrant
+```
+
+2. **kubectl config**
```

Comment:
nit: this doesn't render as "2." -- you will need to indent the code block above.